packages/neuron-wallet/src/services/networks.ts

```diff
@@ -246,8 +246,10 @@ export default class NetworksService extends Store {
         .getBlockchainInfo()
         .then(info => info.chain)
         .catch(() => '')
+    } else if (network.chain === 'ckb_dev') {
+      if (genesisHash === TESTNET_GENESIS_HASH) network.chain = LIGHT_CLIENT_TESTNET
+      if (genesisHash === MAINNET_GENESIS_HASH) network.chain = LIGHT_CLIENT_MAINNET
```

Comment:
we can set the `chain` to `LIGHT_CLIENT_TESTNET` or  `LIGHT_CLIENT_MAINNET`. 
Then https://github.com/nervosnetwork/neuron/pull/3207/files#diff-cd48aaf82ea7f3fdfea4a4bbacc52194bb694fc96a1b5dfc642f098bbc1b10c9R253-R256 will update `network`